website/docs/docs/guides/understanding-state.md

```diff
@@ -69,3 +69,33 @@ The state and result selectors can also be combined in a single invocation of db
 ```bash
 $ dbt run --select result:<status>+ state:modified+ --defer --state ./<dbt-artifact-path>
 ```
+
+### The "source_status" status
+<VersionBlock lastVersion="1.0">
+
+Only supported by v1.1 or newer.
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.1">
+
+Only supported by v1.1 or newer.
+
+:::caution Experimental functionality
+The `source_status` selection method is new in v1.1, and may have rough edges. Give it a try and let us know what you think!
```

Comment:
Let's keep this similar to the cache_selected resource wording. 
```suggestion
The `source_status` selection method should be considered _unstable_  because it is experimental and subject to change. We reserve the right to make breaking changes to this functionality.
```

Reply:
Agreed!